fail2ban/server/filter.py

```diff
@@ -898,22 +898,22 @@ def textToIp(text, useDns):
 		return ipList
 
 	@staticmethod
-	def cidr(i, n):
-		""" Convert an IP address string with a CIDR mask into a 32-bit
-			integer.
+	def netAddr2Dec(ipstring, cidr):
```

Comment:
if anything, we just might want to merge it into add2bin/dec, which would apply cidr if not None.  otherwise -- all other methods are  lower cases...

if you like to improve this -- just RF all the DNSUtils into a separate submodule  (e.g. dnsutils) and remove this unnecessary @static madness... I even have done it at some point -- must be in one of the "lost branches" ;)